hack/util.sh

```diff
@@ -41,13 +41,18 @@ capz::util::should_build_kubernetes() {
     echo "false"
 }
 
+# If should_build_ccm() is true, multi-arch images are built and IMAGE_TAG is the same for CCM and CNM.
+# If false, no images are built. Then If CCM & CNM images are needed, tags should be IMAGE_TAG_CCM & IMAGE_TAG_CNM.
 capz::util::should_build_ccm() {
     # TEST_CCM is an environment variable set by a prow job to indicate that the CCM should be built and tested.
     if [[ -n "${TEST_CCM:-}" ]]; then
         echo "true" && return
     fi
-    # If conformance is being tested with CI artifacts, CCM should be built.
+    # If conformance is being tested with CI artifacts and IMAGE_TAG_CCM or IMAGE_TAG_CNM unset, CCM should be built.
     if [[ "${E2E_ARGS:-}" == "-kubetest.use-ci-artifacts" ]]; then
+        if [[ -n "${IMAGE_TAG_CCM}" && -n "${IMAGE_TAG_CNM}" ]]; then
```

Comment:
I don't think we need to do this, if the images already exist the `can_reuse_artifacts` function will return true and no images will be built or pushed.

Reply:
Updated.